drivers/soundwire/generic_bandwidth_allocation.c

```diff
@@ -206,6 +206,15 @@ static void _sdw_compute_port_params(struct sdw_bus *bus,
 			port_bo = 1;
 
 			list_for_each_entry(m_rt, &bus->m_rt_list, bus_node) {
+				/*
+				 * Only runtimes with CONFIGURED, PREPARED, ENABLED, and DISABLED
+				 * states should be included in the bandwidth calculation.
+				 */
+				if (m_rt->stream->state != SDW_STREAM_CONFIGURED &&
```

Comment:
why not flip the logic and check for permitted states only instead? Also how did you identify this issue? is there a bug filed?

Reply:
Done. And filed the bug: https://github.com/thesofproject/linux/issues/5398